godot-core/src/builtin/projection.rs

```diff
@@ -460,3 +509,474 @@ pub enum ProjectionEye {
+                        0.0,
+                        1.0 / y,
+                        0.0,
+                        0.0,
+                        0.0,
+                        0.0,
+                        1.0 / z,
+                        0.0,
+                        0.0,
+                        0.0,
+                        0.0,
+                        1.0 / w,
+                    ]),
+                    matrix_eq_approx,
+                );
+            }
+        }
+    }
+
+    /// Test `create_orthogonal` method.
+    /// All inputs and outputs are manually computed.
+    #[test]
+    fn test_orthogonal() {
+        const TEST_DATA: [([real; 6], [[real; 4]; 4]); 6] = [
+            (
+                [-1.0, 1.0, -1.0, 1.0, -1.0, 1.0],
+                [
+                    [1.0, 0.0, 0.0, 0.0],
+                    [0.0, 1.0, 0.0, 0.0],
+                    [0.0, 0.0, -1.0, 0.0],
+                    [0.0, 0.0, 0.0, 1.0],
+                ],
+            ),
+            (
+                [0.0, 1.0, 0.0, 1.0, 0.0, 1.0],
+                [
+                    [2.0, 0.0, 0.0, 0.0],
+                    [0.0, 2.0, 0.0, 0.0],
+                    [0.0, 0.0, -2.0, 0.0],
+                    [-1.0, -1.0, -1.0, 1.0],
+                ],
+            ),
+            (
+                [-1.0, 1.0, -1.0, 1.0, 0.0, 1.0],
+                [
+                    [1.0, 0.0, 0.0, 0.0],
+                    [0.0, 1.0, 0.0, 0.0],
+                    [0.0, 0.0, -2.0, 0.0],
+                    [0.0, 0.0, -1.0, 1.0],
+                ],
+            ),
+            (
+                [-10.0, 10.0, -10.0, 10.0, 0.0, 100.0],
+                [
+                    [0.1, 0.0, 0.0, 0.0],
+                    [0.0, 0.1, 0.0, 0.0],
+                    [0.0, 0.0, -0.02, 0.0],
+                    [0.0, 0.0, -1.0, 1.0],
+                ],
+            ),
+            (
+                [-1.0, 1.0, -1.0, 1.0, 1.0, -1.0],
+                [
+                    [1.0, 0.0, 0.0, 0.0],
+                    [0.0, 1.0, 0.0, 0.0],
+                    [0.0, 0.0, 1.0, 0.0],
+                    [0.0, 0.0, 0.0, 1.0],
+                ],
+            ),
+            (
+                [10.0, -10.0, 10.0, -10.0, -10.0, 10.0],
+                [
+                    [-0.1, 0.0, 0.0, 0.0],
+                    [0.0, -0.1, 0.0, 0.0],
+                    [0.0, 0.0, -0.1, 0.0],
+                    [0.0, 0.0, 0.0, 1.0],
+                ],
+            ),
+        ];
+
+        for ([left, right, bottom, top, near, far], mat) in TEST_DATA {
+            assert_eq_approx!(
+                Projection::create_orthogonal(left, right, bottom, top, near, far),
+                RMat4::from_cols_array_2d(&mat),
+                matrix_eq_approx,
+                "left={left} right={right} bottom={bottom} top={top} near={near} far={far}",
+            );
+        }
+    }
+
+    /// Test `create_orthogonal_aspect` method.
+    #[test]
+    fn test_ortho_aspect() {
```

Comment:
There's no reason to abbreviate here, it just makes it harder to associate with the method.
```suggestion
    /// Test `create_orthogonal_aspect` method.
    #[test]
    fn test_orthogonal_aspect() {
```